app/parse.py

```diff
@@ -1,9 +1,42 @@
-from dataclasses import dataclass
+import csv
+from dataclasses import dataclass, astuple, fields
+from time import sleep
 from urllib.parse import urljoin
 
+from selenium.common import NoSuchElementException
+from selenium import webdriver
+from selenium.webdriver.firefox.webdriver import WebDriver
+from selenium.webdriver.common.by import By
+from selenium.webdriver.remote.webelement import WebElement
 
 BASE_URL = "https://webscraper.io/"
 HOME_URL = urljoin(BASE_URL, "test-sites/e-commerce/more/")
+COMPUTERS_URL = urljoin(BASE_URL, "test-sites/e-commerce/more/computers")
+LAPTOPS_URL = urljoin(BASE_URL, "test-sites/e-commerce/more/computers/laptops")
+TABLETS_URL = urljoin(BASE_URL, "test-sites/e-commerce/more/computers/tablets")
+PHONES_URL = urljoin(BASE_URL, "test-sites/e-commerce/more/phones")
+TOUCH_URL = urljoin(BASE_URL, "test-sites/e-commerce/more/phones/touch")
+
+URLS_AND_FILES = {
+    HOME_URL: "home.csv",
+    COMPUTERS_URL: "computers.csv",
+    LAPTOPS_URL: "laptops.csv",
+    TABLETS_URL: "tablets.csv",
+    PHONES_URL: "phones.csv",
+    TOUCH_URL: "touch.csv"
+}
+
+
+_driver: WebDriver | None = None
+
+
+def get_driver() -> WebDriver:
+    return _driver
+
+
+def set_driver(new_driver: WebDriver) -> None:
+    global _driver
+    _driver = new_driver
```

Comment:
I think you are complicating it here, you can implement it all as a class, implement the `init_browser_instances` method in the class